Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```diff
@@ -27,6 +27,13 @@ td.tabbedMainMenuCell
   background-color: var(--color-contrast);
 }
 
+div.tabbedMainMenu div.tabStripTabsPane ul li,
+div.tabbedSubMenu div.tabStripTabsPane ul li
+{
+  flex-grow: unset;
+  justify-content: start;
```

Comment:
Something got lost in translation here. The 'justify' needs to be applied to another selector (see ticket). let's do it together.